Formula/a/apitrace.rb

```diff
@@ -0,0 +1,133 @@
+class Apitrace < Formula
```

Comment:
This whole formula fails `brew style apitrace.rb` because I only saw that command after submitting this PR.
I'll address this soon(tm).